### ISSUE TITLE
dynamic placement of LoF and LoT?

### ISSUE THREAD

**Issue by andreifoldes** [ismayc/thesisdown#200]
**Describe the bug**
Greetings,
I was hoping to place the LoT and LoF at the back of my thesis instead of the front?

**To Reproduce**
 I tried writing \listoffigures and \listoftables at the end but I didn't appear to work.

**Comment by ismayc** [comment]
Hi there. Just to confirm, you tried moving those LaTeX commands in the template LaTeX file and it still didn’t work? Or where did you move the commands to?

**Comment by andreifoldes** [comment]
yes, I tried moving them both in the rmarkdown file and in the post-knit .tex file, but in the first case one would get duplicates in the latter case i just got empty list ...

**Comment by ismayc** [comment]
I was referring to the template.tex file, not the post-knit file. Are you using your own template file or the one provided here?
